04-Multi-Band-Raster.Rmd

```diff
@@ -70,22 +65,36 @@ Read more about the `raster` package in R.</a>
 
 #About Raster Bands
 
-As discussed in the [Intro to Raster Data Lesson 00]( {{ base.url }} }}/R/Introduction-to-Raster-Data-In-R), a raster can contain 1 or more bands. To 
-work with multi-band rasters, we need to adjust how we import and plot our data. 
+As discussed in the [Intro to Raster Data Lesson 00]( {{ base.url }} }}NEON-R-Spatial-Raster/R/Introduction-to-Raster-Data-In-R), a raster can
+contain 1 or more bands. To work with multi-band rasters, we need to change how 
+we import and plot our data in several ways. 
 
-* To import multi band raster data we will use the `stack` function.
-* If our multi-band data are imagery that we wish to composit, we can use `plotRGB` 
-to plot a 3 band image raster.
+* To import multi band raster data we will use the `stack()` function.
+* If our multi-band data are imagery that we wish to composite (spatial data
+term essentially meaning, to view multiple bands together), we can use
+`plotRGB()` to plot a 3 band image raster.
+
+>Bonus: Bands? Layers?  Why are both names used? The term bands comes from the 
```

Comment:
How about 1)Change all layers to bands except were we really mean layer.  2) Substantially shorten this bonus.  3) Add in the Mapping The Invisible video to the bonus bit: https://www.youtube.com/watch?v=3iaFzafWJQE&noredirect=1.